cmd/arcaflow/main.go

```diff
@@ -57,53 +58,42 @@ func main() {
 	workflowFile := "workflow.yaml"
 	printVersion := false
 
-	flag.BoolVar(&printVersion, "version", printVersion, "Print Arcaflow Engine version and exit.")
-	flag.StringVar(
-		&configFile,
-		"config",
-		configFile,
-		"The Arcaflow configuration file to load, if any.",
-	)
-	flag.StringVar(
-		&input,
-		"input",
-		input,
-		"The workflow input file to load. May be outside the workflow directory. If no input file is passed, "+
-			"the workflow is assumed to take no input.",
-	)
-	flag.StringVar(
-		&dir,
-		"context",
-		dir,
-		"The workflow directory to run from. Defaults to the current directory.",
-	)
-	flag.StringVar(
-		&workflowFile,
-		"workflow",
-		workflowFile,
-		"The workflow file in the current directory to load. Defaults to workflow.yaml.",
+	const (
+		helpUsage    = "Display this help output."
+		versionUsage = "Print Arcaflow Engine version and exit."
+		configUsage  = "The Arcaflow configuration file to load, if any."
+		inputUsage   = `The workflow input file to load. May be outside
+                           the workflow directory. If no input file is
+                           passed, the workflow is assumed to take no input.`
+		contextUsage = `The workflow directory to run from. Defaults 
+                           to the current directory.`
+		workflowUsage = `The workflow file in the current directory to load.
+                           Defaults to workflow.yaml.`
 	)
+	flag.BoolVar(&printVersion, "version", printVersion, versionUsage)
+	flag.BoolVar(&printVersion, "v", printVersion, versionUsage+" (shorthand)")
+	flag.StringVar(&configFile, "config", configFile, configUsage)
+	flag.StringVar(&configFile, "c", configFile, configUsage+" (shorthand)")
+	flag.StringVar(&input, "input", input, inputUsage)
+	flag.StringVar(&input, "i", input, inputUsage+" (shorthand)")
+	flag.StringVar(&dir, "context", dir, contextUsage)
+	flag.StringVar(&dir, "t", dir, contextUsage+" (shorthand)")
```

Comment:
Other than the fact that we can't use `c` because it's already in use, is there a reason `t` is used for context? Unless there's a reason for this, I think `x` would make more sense.

Reply:
`t` was just the first alternative I thought of. I'm happy to change it to `x` instead.